terraform/terraform.tfvars

```diff
@@ -6,14 +6,19 @@ sql_server_name     = "sql-server-apprenticeship"
 nsg_name            = "nsg"
 vnet_name           = "vnet"
 address_space       = ["10.0.0.0/16"]
-subnets = [
+
+networks = [
```

Comment:
De ce am creat acest map? Asta trebuie numit subnet, mie network nu imi zice mare lucru cand ma uit la cod.
Din ce inteleg ai incercat sa combini crearea de IPs si subnets intr-un singur map, nu este corect pentru ca IP-urile nu sunt legate de subnet-uri, sunt legate de VM-uri, poti sa ai trei VM-uri intr-un subnet toate cu IP-ul lor.
In alte cuvinte aici ne trebuie doua map-uri, unul pentru subnet, unul pentru IP

Reply:
Pe NIC IPv4 trebuie obligatoriu si subnet la [legarea de NIC](https://github.com/EndavaCollab/bvdapprenticeship2024-infra/actions/runs/10490511105/job/29057403322#step:8:350) a unui IP public dar le-am separat ca sa fie posibil si un subnet fara IP public legat de el.